app/forms/candidate_interface/science_gcse_grade_form.rb

```diff
@@ -203,7 +203,11 @@ def grade_hash(grade)
     end
 
     def sanitize(grade)
-      grade&.delete(' ')&.upcase
+      if DOUBLE_GCSE_GRADES.exclude?(grade)
+        grade&.delete(' ')&.delete('/')&.delete(',')&.delete('-')&.upcase
```

Comment:
You could do sth like `grade.gsub(/\W/,'')` to remove all non-alphanumeric chars?

Reply:
You might want `[^*\w]` if you need to preserve `*` characters?

Reply:
@duncanjbrown so the problem with `grade.gsub(/\W/,'')` is that it now affects sanitized grades with a `*` in them e.g. `a*a` becomes sanitized to 'AA'. 

Is there a way to exclude `*` from the xpression?

Reply:
Just as I asked 😆